AltStore/Info.plist

```diff
@@ -90,7 +90,7 @@
 		<string>_altserver._tcp</string>
 	</array>
 	<key>NSLocalNetworkUsageDescription</key>
-	<string>SideStore uses the local network to find and communicate with SideServer.</string>
+	<string>SideStore uses the local network to find and communicate with your device.</string>
```

Comment:
Pretty sure this whole entitlement can just be removed

Reply:
The latest Commit should resolve this issue.